packages/@aws-cdk/aws-lambda/lib/layers.ts

```diff
@@ -3,21 +3,10 @@ import { Code } from './code';
 import { CfnLayerVersion, CfnLayerVersionPermission } from './lambda.generated';
 import { Runtime } from './runtime';
 
-export interface LayerVersionProps {
-  /**
-   * The runtimes compatible with this Layer.
-   *
-   * @default - All runtimes are supported.
-   */
-  readonly compatibleRuntimes?: Runtime[];
-
-  /**
-   * The content of this Layer.
-   *
-   * Using `Code.fromInline` is not supported.
-   */
-  readonly code: Code;
-
+/**
+ * Non runtime options
+ */
+export interface LayerVersionOptions {
   /**
```

Comment:
Leaving these outstanding questions here for feedback:

1. Should this be `LayerVersionBaseProps` and can it also include `compatibleRuntimes`?